ansible-advancedlinux/azuredeploy.json

```diff
@@ -123,6 +67,39 @@
 				"description": "Servers role, for instance webtier, database.A tag will be created with the provided value"
 			}
 		},
+		"vmNamePattern": {
+			"type": "string",
+			"defaultValue": "VMCba",
```

Comment:
What is "cba"? It appears many places. In general avoid abbreviations that aren't common